control/autoware_autonomous_emergency_braking/README.md

```diff
@@ -148,19 +162,19 @@ $$
 
 where $yaw_{diff}$ is the difference in yaw between the ego path and the displacement vector $$v_{pos} = o_{pos} - prev_{pos} $$ and $v_{ego}$ is the ego's current speed, which accounts for the movement of points caused by the ego moving and not the object. All these equations are performed disregarding the z axis (in 2D).
 
-Note that, the object velocity is calculated against the ego's current movement direction. If the object moves in the opposite direction to the ego's movement, the object velocity will be negative, which will reduce the rss distance on the next step.
+Note that the object velocity is calculated against the ego's current movement direction. If the object moves in the opposite direction to the ego's movement, the object velocity will be negative, which will reduce the rss distance on the next step.
 
 The resulting estimated object speed is added to a queue of speeds with timestamps. The AEB then checks for expiration of past speed estimations and eliminates expired speed measurements from the queue, the object expiration is determined by checking if the time elapsed since the speed was first added to the queue is larger than the parameter `previous_obstacle_keep_time`. Finally, the median speed of the queue is calculated. The median speed will be used to calculate the RSS distance used for collision checking.
 
 ### 5. Collision check with target obstacles using RSS distance
 
-In the fourth step, it checks the collision with the closest obstacle point using RSS distance. RSS distance is formulated as:
+In the fifth step, it checks the collision with the closest obstacle point using RSS distance. RSS distance is formulated as:
 
 $$
 d = v_{ego}*t_{response} + v_{ego}^2/(2*a_{min}) -(sign(v_{obj})) * v_{obj}^2/(2*a_{obj_{min}}) + offset
 $$
 
-where $v_{ego}$ and $v_{obj}$ is current ego and obstacle velocity, $a_{min}$ and $a_{obj_{min}}$ is ego and object minimum acceleration (maximum deceleration), $t_{response}$ is response time of the ego vehicle to start deceleration. Therefore the distance from the ego vehicle to the obstacle is smaller than this RSS distance $d$, the ego vehicle send emergency stop signals. This is illustrated in the following picture.
+where $v_{ego}$ and $v_{obj}$ is current ego and obstacle velocity, $a_{min}$ and $a_{obj_{min}}$ is ego and object minimum acceleration (maximum deceleration), $t_{response}$ is response time of the ego vehicle to start deceleration. Therefore the distance from the ego vehicle to the obstacle is smaller than this RSS distance $d$, the ego vehicle send emergency stop signals. Note that only "target" obstacles, as defined on step #3 are considered for RSS distance calculation. If the closest obstacle is not a "target" (its not inside the predicted ego path defined by the ego vehicle width and the expand margin), this step is skipped, and not emergency stop diagnostic message is sent. This process is illustrated in the following picture.
```

Comment:
@danielsanchezaran 
>If the closest obstacle is not a "target" (its not inside the predicted ego path defined by the ego vehicle width and the expand margin), this step is skipped, and not emergency stop diagnostic message is sent.

I thought that among the point clouds within the range of 'vehicle width + expand_width', it would select the nearest point cloud that has 'is_target' set to true as the nearest neighbor. Am I mistaken?

Reply:
fixed